renderdoc/driver/ihv/nv/nv_counters.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <stdint.h>
+#include <vector>
+#include "api/replay/renderdoc_replay.h"
+
+struct ID3D11Device;
+
+class NVCounters
+{
+public:
+  NVCounters();
+  ~NVCounters();
+
+  bool Init(ID3D11Device *pDevice);
+
+  std::vector<GPUCounter> GetPublicCounterIds() const { return mExternalIds; }
+  CounterDescription GetCounterDescription(GPUCounter counterID) const
+  {
+    const uint32_t LocalId =
+        static_cast<uint32_t>(counterID) - static_cast<uint32_t>(GPUCounter::FirstNvidia);
+    return mExternalDescriptors[LocalId];
+  }
+
+  bool PrepareExperiment(const std::vector<GPUCounter> &counters, uint32_t objectsCount);
+
+  // returns num passes
+  uint32_t BeginExperiment() const;
+  void EndExperiment(const std::vector<uint32_t> &eventIds, std::vector<CounterResult> &Result) const;
+
+  void BeginPass(uint32_t passIdx) const;
+  void EndPass(uint32_t passIdx) const;
+
+  void BeginSample(uint32_t sampleIdx) const;
+  void EndSample(uint32_t sampleIdx) const;
+
+private:
+  bool Init(void);
+
+  void *mNvPmLib;
```

Comment:
For members either use `m_` prefix or not at all, the plain `m` prefix is not used anywhere else.